luskan/src/main/resources/sql/tap_schema_content11.sql

```diff
@@ -0,0 +1,65 @@
+-- delete key columns for keys from tables in the caom2 schema
+delete from tap_schema.key_columns11 where
+key_id in (select key_id from tap_schema.keys11 where
+    from_table in (select table_name from tap_schema.tables11 where schema_name = 'inventory')
+    or
+    target_table in (select table_name from tap_schema.tables11 where schema_name = 'inventory')
+)
+;
+
+-- delete keys from tables in the ivoa schema
+delete from tap_schema.keys11 where
+from_table in (select table_name from tap_schema.tables11 where schema_name = 'inventory')
+or
+target_table in (select table_name from tap_schema.tables11 where schema_name = 'inventory')
+;
+
+-- delete columns from tables in the ivoa schema
+delete from tap_schema.columns11 where table_name in
+(select table_name from tap_schema.tables11 where schema_name = 'inventory')
+;
+
+-- delete tables
+delete from tap_schema.tables11 where schema_name = 'inventory'
+;
+
+-- delete the schema
+delete from tap_schema.schemas11 where schema_name = 'inventory'
+;
+
+insert into tap_schema.schemas11 (schema_name,description) values
+('inventory', 'tables and views used in the inventory system')
+;
+
+-- index start at 10
+insert into tap_schema.tables11 (schema_name,table_name,table_type,description,table_index) values
```

Comment:
missing inventory.StorageSite

Reply:
I thought that it's not part of the schema.